internal/backend/db/dbgorm/dbtime/dbtime.go

```diff
@@ -0,0 +1,13 @@
+package dbtime
```

Comment:
You can merge this as-is, but in my opinion it's misleading to call this "DB time" when the purpose is to enable testing with fake time, and the usability of this utility extends beyond the DB.

Reply:
will move to kittehs